### PR TITLE
Fix GitHub actions

### PR DESCRIPTION
**Pull request by ruudk**
* Run Action on commit
* Split tests between master / pull requests
* Remove ci-trigger make target
* Replace status badge
* Publish coverage on master

### CODE REVIEW

**Comment by ruudk** [comment]
When https://github.com/zalando/skipper/pull/1402 is merged, I will rebase this.

File: .github/workflows/master.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.14'
+    - run: go version
+    - run: sudo apt-get install redis-server
+    - run: make deps check-race cicheck check-plugins
```
Comment (aryszka) [inline .github/workflows/master.yaml:18]: maybe we can add check-plugins as separate step, and remove the rest together with the condition?

**Comment by aryszka** [comment]
I am fine with addressing publish-coverage in a separate step

**Comment by ruudk** [comment]
What’s the goal of the publish coverage? Should it only run on the last tagged release? Or can we just run it on master, kind of the same right?

**Comment by aryszka** [comment]
The "goal" goal of the publish_coverage is to motivate us to improve the actual test coverage by showing an approximation ;)

for real, its goal is to show the test coverage of the master branch. So it's enough to run it after changes were pushed to master. But we don't want to make it a condition of the PRs.

**Comment by aryszka** [comment]
:+1:

**Comment by ruudk** [comment]
Oke then I understand it correctly, will update the PR and try to make it green ;-)

File: .github/workflows/master.yaml (hunk 1 above)
Comment (ruudk) [reply .github/workflows/master.yaml]: I re-arranged it and forced pushed. Can you reply on the `master.yaml` file which steps I should remove? Then I'll mark this comment as resolved.

**Comment by ruudk** [comment]
@szuecs @aryszka It's green 🎉

**Comment by aryszka** [comment]
:+1:

**Comment by ruudk** [comment]
Please merge this and make these required checks:

* pr / check-race (pull_request)
* pr / tests (pull_request)

**Comment by muaazsaleem** [comment]
👍

**Pull request merged**